File: APIRollTheDice-Backend/src/main/java/com/example/APIRollTheDice/Model/DTO/Chat/ChatChanelDTO.java
package com.example.APIRollTheDice.Model.DTO.Chat;


import java.util.List;
import java.util.Set;

public class ChatChanelDTO {
    private Long id;

    private String name;

    private List<Long> idMessages;

    private Long idGame;



    public ChatChanelDTO() {
    }

    public ChatChanelDTO(Long id, String name, List<Long> idMessages, Long idGame) {
        this.id = id;
        this.name = name;
        this.idMessages = idMessages;
        this.idGame = idGame;
    }

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public List<Long> getIdMessages() {
        return idMessages;
    }

    public void setIdMessages(List<Long> idMessages) {
        this.idMessages = idMessages;
    }

    public Long getIdGame() {
        return idGame;
    }

    public void setIdGame(Long idGame) {
        this.idGame = idGame;
    }
}

File: RollTheDice/Assets/_Project/API/Model/DTO/ChatDTO/ChatChanelDTO.cs
using System.Collections.Generic;

namespace Assets._Project.API.Model.DTO.ChatDTO
{
    public class ChatChanelDTO
    {
        public long Id { get; set; }

        public string Name { get; set; }

        public List<long> IdMessages { get; set; }

        public long IdGame { get; set; }

        public ChatChanelDTO(){}
        public ChatChanelDTO(long id,string name, List<long> idMessage,long idGame)
        {
            Id = id;
            Name = name;
            IdMessages = idMessage;
            IdGame = idGame;
        }
    }
}